source/includes/edge/limitations.rst

```diff
@@ -0,0 +1,26 @@
+While Edge Server is in Public Preview, it does not yet support the full range
+of MongoDB and Atlas Device SDK features and functionality.
+
+Collection Types
+````````````````
+
+Edge Server does not currently support time series collections.
```

Comment:
do we want to add a link to the [time series](https://www.mongodb.com/docs/v6.0/core/timeseries-collections/) doc?